src/app/containers/Video/index.jsx

```diff
@@ -16,9 +17,32 @@ const VideoContainer = ({ blocks }) => {
     return null;
   }
 
+  const nestedModel = deepGet(['model', 'blocks', 0, 'model'], aresMediaBlock);
+  const kind = deepGet(['subType'], nestedModel);
+  const pid = deepGet(['id'], nestedModel);
+  const title = deepGet(['title'], nestedModel);
+  const version = deepGet(['versions', 0], nestedModel);
+  const duration = deepGet(['duration'], version);
+  const versionID = deepGet(['versionId'], version);
+  const holdingImageUrl = deepGet(
+    ['blocks', 1, 'model', 'blocks', 0, 'model', 'locator'],
+    aresMediaBlock.model,
+  );
+  const items = {
+    versionID,
+    kind,
+    duration,
+  };
+
   return (
     <Figure>
-      <Video />
+      <Video
+        pid={pid}
+        kind={kind}
```

Comment:
I think kind is most likely going to be “type” in all the AV APIs so might be good to remain consistent

Reply:
Kind is what the media player uses, so that is why it is defined that way here.